### ISSUE TITLE
Plot improvements

### ISSUE THREAD

**Issue by vadimyer** [ForgeEssentials/ForgeEssentials#1873]
I know there are already some threads about plots functionality, and you probably want to make them more original and unique than a mere "private regions bukkit plugin". I've recently found FE and have to say that's an incredible work. You've wrote that it might seem to work slower than Cauldron, but to be honest I've discovered FE to be much faster in terms of performance, after I was tired from lags and freezes with a brand new world on KCauldron. My server now runs smoothly on FE, without any lags at all. That's like heaven and earth!

But I have a few questions:
- Plots are a real deal, but they lack (as well as areas) some ability to set default configuration. Could it be in the nearest future? And is it appropriate to write a script for this kind of task right now? Like, player Bruce created a plot, and it automatically added some of the perms and some custom perms for his plot.
- BTW, villagers can be hurt, pressure plates can be used, some unusual (from some mods, but not from all) doors can be opened, but from inside (if opening them from outside though, a player can't enter inside). Those are things I'd like to know a perfect perm for.

Thanks.

**Comment by olee** [comment]
You are right, there are a lot of features that are not quite as finished as I wish them to be.
However we are kinda lacking developers which makes work really difficult.
I will try to get to this if I have time...

**Comment by vadimyer** [comment]
If I were into Java, would certainly help with all the work. Thanks for the great work.
BTW, can you explain how pregen works? I set world borders to 10000x10000 and started pregen, after a while it finished at 1200000/1500000, like 300k were missing or something. And I then tried to teleport to different areas in a border radius, but they were loading pretty slowly for the first time. Then I set the warps at those points and left the server for a while, came back and starting teleporting to those warps and they were loading very quickly, like at that moment - they were indeed pregenerated. Maybe I don't understand something in world generation?

**Comment by vadimyer** [comment]
@olee Please can you explain how do one write a script that for example sets users group after he executed /auth register <password>? I can't get it to work with commands (I created a folder called "auth" and player.txt inside it, with commands to execute like in the tutorial). Wel after all, setting default permissions after plot define/registration are some most essential things, I try to achieve that with scripts but have problems.
